project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.11")
-addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.9")
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.18")
```

Comment:
Updating to get some nice defaults - https://github.com/guardian/sbt-riffraff-artifact/pull/63.

Reply:
Presume this does not affect this project? I can't see any values removed in this PR.